src/Installer/finalizer/finalizer.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(SdkTargetFramework)-windows</TargetFramework>
+    <OutputType>Exe</OutputType>
```

Comment:
This can be changed to WinExe, we don't want console windows opening up at all - it needs to run 100% silently